src/ApiCall.ts

```diff
@@ -41,7 +45,6 @@ export async function getConfig() {
 }
 
 function getHostEndPoints(config: AppConfig) {
-  consoleLog(config, 'API call - getHostEndPoints');
```

Comment:
Can we actually leave all this logging in? I didn't realise it was only used in staging and not prod so won't be an issue for penetration testing, I can argue against that